# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/heap-analysis/findPath.js
# Commit: 22854b710fbf
# Full Hash: 22854b710fbfb779ac6b19fe4c9ac2570189cec0
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 4531d5c25694 -r 22854b710fbf js/src/jit-test/tests/heap-analysis/findPath.js
--- a/js/src/jit-test/tests/heap-analysis/findPath.js	Fri Mar 05 19:10:06 2021 +0000
+++ b/js/src/jit-test/tests/heap-analysis/findPath.js	Fri Mar 05 19:10:07 2021 +0000
@@ -35,8 +35,9 @@
   .assert(findPath(gc, o));
 print(JSON.stringify(findPath(gc, o)));
 
-Match.Pattern([{node: {}, edge: "group"},
-               {node: Match.Pattern.ANY, edge: "group_global"},
+Match.Pattern([{node: {}, edge: "shape"},
+               {node: Match.Pattern.ANY, edge: "base"},
+               {node: Match.Pattern.ANY, edge: "baseshape_global"},
                {node: {}, edge: "o"}])
   .assert(findPath(o, o));
 print(findPath(o, o).map((e) => e.edge).toString());